integration-tests/netty/src/main/java/org/apache/camel/quarkus/component/netty/CamelRoute.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.netty;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.apache.camel.builder.RouteBuilder;
+
+public class CamelRoute extends RouteBuilder {
+
+    @Override
+    public void configure() {
+
+            from("netty:tcp://localhost:8999?textline=true&sync=true")
+            .process(new Processor() {
+                public void process(Exchange exchange) throws Exception {
+                    if (exchange.getIn().getBody() instanceof Poetry) {
+                        Poetry poetry = (Poetry) exchange.getIn().getBody();
```

Comment:
This conditional branch is actually not covered by the test. I wonder whether we should add another test covering it or remove this branch (and the Poetry class)? WDYT Camelers?

Reply:
I do not disagree there could be additional tests to handle the code-coverage, however recommend accepting this PR and allow additional tests from future PR's.